pages.hi/windows/winget.md

```diff
@@ -0,0 +1,36 @@
+# winget
+
+> विंडोज़ पैकेज प्रबंधक।
+> अधिक जानकारी: <https://learn.microsoft.com/windows/package-manager/winget>.
+
+- एक पैकेज इनस्टॉल करें:
+
+`winget install {{package}}`
+
+- एक पैकेज निकालें (अनइंस्टॉल करें): (नोट: `uninstall` की जगह `remove` का भी इस्तेमाल किया जा सकता है):
+
+`winget uninstall {{package}}`
+
+- पैकेज के बारे में जानकारी प्रदर्शित करें:
+
+`winget show {{package}}`
+
+- पैकेज की खोज करें:
+
+`winget search {{package}}`
+
+- सभी पैकेज़ को नवीनतम संस्करणों में अपग्रेड करें:
+
+`winget upgrade --all`
+
+- `winget` के साथ प्रबंधित इन्सटाल्ड पैकेजों की सूची बनाएं:
+
+`winget list --source winget`
+
+- किसी फ़ाइल से पैकेज आयात करें, या स्थापित पैकेज़ को किसी फ़ाइल में निर्यात करें:
+
+`winget {{import|export}} {{--import-file|--output}} {{path/to/file}}`
+
+- विंगेट-पीकेजीएस(winget-pkgs) रिपॉजिटरी में पीआर(PR) सबमिट करने से पहले मैनिफ़ेस्ट को सत्यापित करें:
+
+`winget validate {{path/to/manifest}}`
```

Comment:
Please translate path/to/manifest as well.

Reply:
```suggestion
`winget validate {{प्रकट/का/पथ}}`
```
Feel free to reword "manifest".